qt/applications/workbench/workbench/plotting/figuremanager.py

```diff
@@ -283,6 +285,30 @@ def destroy(self, *args):
             # Gcf can get destroyed before the Gcf.destroy
             # line is run, leading to a useless AttributeError.
 
+    def on_home_clicked(self):
+        """
+        The matplotlib home button centres and scales the figure.
+        When we have visible errorbars, we should scale off of the bounds
+        of the error bars.
+        """
+        ax = self.canvas.figure.axes[0]
+        creation_args = ax.creation_args
+
+        min_bounds = []
+        max_bounds = []
+        for cargs in creation_args:
+            # Creation args do not necessarily contain POST_CREATION_ARGS
+            if (MantidAxKwargs.POST_CREATION_ARGS in cargs and
+                    cargs[MantidAxKwargs.POST_CREATION_ARGS][MantidAxKwargs.ERRORS_VISIBLE]):
+                for container in ax.containers:
+                    caps = container[1]
+                    if caps != ():
+                        min_bounds.extend(caps[0].get_ydata())
```

Comment:
I think these may be `numpy.ndarray`. If so they should have `.min()`/`.max()` methods